src/com/sap/piper/tools/ToolExecutable.groovy

```diff
@@ -0,0 +1,26 @@
+package com.sap.piper.tools
+
+
+enum ToolExecutable {
+
+    JAVA ('/bin/java', ''), 
```

Comment:
The reason is that if the executable is in the PATH, it does not need the ToolExecutable path, just the ToolExecutable name, Otherwise, it needs both.

Reply:
Following your pattern - shouldn't it be ```JAVA('/bin/', 'java')```in this case?